### ISSUE TITLE
ADA: Color contrast does not meet minimum requirement

### ISSUE THREAD

**Issue by rbruhn** [AustinMastLab/ckanext-digitizationknowledge#66]
There is insufficient color contrast between the text and background on this page to meet the minimum contrast requirement.

**Comment by rbruhn** [comment]
Fixed with 02465c01f0cdd4e4cfcf4907c0c417da4ca767bf
Verify after crawl.

**Comment by rbruhn** [comment]
Tests complete and showing 100%
